BoxContentSDK/BoxContentSDK/Operations/BOXAPIMultipartToJSONOperation.m

```diff
@@ -21,454 +20,130 @@
 #pragma mark - Form Boundary Helpers
 
 static NSString *const BOXAPIMultipartFormBoundary = @"0xBoXSdKMulTiPaRtFoRmBoUnDaRy";
-static NSString *const BOXAPIMultipartFormCRLF     = @"\r\n";
-
-static NSString * BOXAPIMultipartInitialBoundary(void)
-{
-    return [NSString stringWithFormat:@"--%@%@", BOXAPIMultipartFormBoundary, BOXAPIMultipartFormCRLF];
-}
-
-static NSString * BOXAPIMultipartEncapsulationBoundary(void)
-{
-    return [NSString stringWithFormat:@"%@--%@%@", BOXAPIMultipartFormCRLF, BOXAPIMultipartFormBoundary, BOXAPIMultipartFormCRLF];
-}
-
-static NSString * BOXAPIMultipartFinalBoundary(void)
-{
-    return [NSString stringWithFormat:@"%@--%@--%@", BOXAPIMultipartFormCRLF, BOXAPIMultipartFormBoundary, BOXAPIMultipartFormCRLF];
-}
 
 static NSString * BOXAPIMultipartContentTypeHeader(void)
 {
     return [NSString stringWithFormat:@"multipart/form-data; boundary=%@", BOXAPIMultipartFormBoundary];
 }
 
-@implementation BOXAPIMultipartPiece
 
-@synthesize headers = _headers;
-@synthesize headersInputStream = _headersInputStream;
-@synthesize bodyInputStream = _bodyInputStream;
-@synthesize bodyContentLength = _bodyContentLength;
-@synthesize hasInitialBoundary = _hasInitialBoundary;
-@synthesize startBoundaryInputStream = _startBoundaryInputStream;
-@synthesize hasFinalBoundary = _hasFinalBoundary;
-@synthesize endBoundaryInputStream = _endBoundaryInputStream;
-@synthesize state = _state;
+#pragma mark - Upload Operation
 
-- (id)initWithData:(NSData *)data fieldName:(NSString *)fieldName filename:(NSString *)filename
+@interface BOXAPIMultipartToJSONOperation ()
 {
-    self = [self initWithData:data fieldName:fieldName filename:filename MIMEType:nil];
-
-    return self;
+    dispatch_once_t _pred;
```

Comment:
See the recent PR (https://github.com/box/box-ios-sdk/pull/316), we need to take this out.